gto/git_utils.py

```diff
@@ -67,3 +67,17 @@ def cloned_git_repo(repo: str):
 def git_clone(repo: str, dir: str) -> None:
     logging.debug("clone %s in directory %s", repo, dir)
     Repo.clone_from(url=repo, to_path=dir)
+
+
+def git_push_tag(repo_path: str, tag_name: str, remote_name: str = "origin") -> None:
+    repo = Repo(path=repo_path)
+    remote = repo.remote(name=remote_name)
+    remote_url = "unknown" if not hasattr(remote, "url") else remote.url
+    logging.debug(
+        "push tag %s from directory %s to remote %s with url %s",
+        tag_name,
+        repo_path,
+        remote_name,
+        remote_url,
+    )
+    remote.push(tag_name)
```

Comment:
that's what I would guess too, why?

Reply:
Sorry, didn't get the question :)
I assume this would fail cause you invoke `push`, but `remote_url` is None, so it don't know where to push to. Am I wrong?

Reply:
`remote_url` is just used in the logging (just a safe wrapper around `remote.url`), I don't think it has nothing to do with failure of `remote.push`.

But I think you are onto something...?

Reply:
ok, I was trying to check if there are cases when you actually get a remote without URL (`remote_url == "unknown"` eventually), in this case `remote.push` could fail (because there is no actual remote to use). In this case it would be good to throw some GTO error (like `WrongArg` for example), and not a GitPython one.